.dockerignore

```diff
@@ -0,0 +1,4 @@
+.gitignore
+LICENSE
+README.md
+manifest.json
```

Comment:
I think you need to add node_modules to this file, otherwise, your `npm install` isn't going to do anything (and could cause problems if you are developing on a mac/windows and have binary modules)